src/sailor/db/elasticsearch.lua

```diff
@@ -0,0 +1,132 @@
+-------------------------------------------------------------------------------------------
+-- elasticsearch.lua: DB module for connecting, querying and serching through elasticsearch
+-- This file is a part of Sailor project
+-- Copyright (c) 2016 Etiene Dalcol <dalcol@etiene.net>
```

Comment:
you can put your own identification on this file because it was created by you :)